src/tasks/task1.js

```diff
@@ -0,0 +1,5 @@
+module.exports = (arrayClothes, param, value) => {
+  const result = arrayClothes.filter(clothes => clothes[param] === value);
+
+  return result;
```

Comment:
`return arrayClothes.filter(clothes => clothes[param] === value);` - one line.

Reply:
If it is not use, `result`, i can immediately return it?

Reply:
Everything can be returned.
In this case there is a result and you may store it to a variable and return that variable or you may return the result right away.